dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassFile.java

```diff
@@ -26,11 +26,11 @@
  * A tuple of a class name and a jar file to uniquely locate the class implementation in a class
  * path.
  */
-final class ClassFile {
+public final class ClassFile {
```

Comment:
Perhaps not in the cl, but maybe the right name for this is "CompiledClass" or something like that. In ther words this uniquely locates the byte code of a Java class using a class name and a jar path.

Reply:
I feel `ClassFile` is still good because a class file in Java means it's compiled.

To clarify it, updated javadoc to `A compiled class file of {@code className} in {@code jar}...`.